openmc/source.py

```diff
@@ -104,6 +104,14 @@ def strength(self, strength):
         self._strength = strength
 
     def to_xml(self):
+        """Return XML representation of the source
```

Comment:
I've been thinking about this for a while too. My preferred convention would be to have `to_*` and `from_*` methods as indicating conversion from one type in memory to another. So in this case, a more proper method name might be `to_xml_element()`. For writing/reading from a file, I'd prefer `write_to*` and `read_from_*`, or alternatively `export_to_*` and `import_from_*`. I think this makes it clearer as to when you should expect an object to be returned a method and when you just want to perform disk IO.